static/js/biomedical/disease/page.tsx

```diff
@@ -87,7 +92,15 @@ export class Page extends React.Component<PagePropType, PageStateType> {
       { id: "name", name: "Compound Name" },
       { id: "drugSource", name: "Drug Source" },
     ];
-
+    const diseasePrevalenceStatVarValue =
+      "Count_MedicalConditionIncident_Condition" + this.props.dcid;
+    const statVarDisease = {} as StatVarSpec;
```

Comment:
instead of defining an empty object and then setting the fields, you could just define the object with all the values like this:

```
const statVarSpec = {
statVar: diseasePrevalenceStatVarValue,
denom: "",
...
}
```

Reply:
Done!